scripts/globals/abyssea.lua

```diff
@@ -1206,14 +1206,11 @@ xi.abyssea.searingWardTimer = function(player)
     local tetherTimer = player:getLocalVar('tetherTimer')
 
     if tetherTimer > 1 then
-        if tetherTimer == 11 or tetherTimer <= 6 then
+        player:setLocalVar('tetherTimer', tetherTimer - 1)
+
+        if tetherTimer <= 6 then
             player:messageSpecial(ID.text.RETURNING_TO_SEARING_IN, tetherTimer - 1)
         end
-
-        player:setLocalVar('tetherTimer', tetherTimer - 1)
-        player:timer(1500, function()
```

Comment:
By removing this timer, you're relying on the effect tick of 3s as opposed to 1.5s.  I believe that makes this countdown much longer in that case (if I remember correctly, I went with 1.5s due to timing as well between client and server)

Reply:
Yea I realized that a bit late, originally I thought for some reason they had similar tick times and were redundant, but later realized in a comment my mistake.

I should be able to just drop that commit and then it'll just be the message fix.